docs/source/conf.py

```diff
@@ -219,6 +220,16 @@
 # Output file base name for HTML help builder.
 htmlhelp_basename = 'StackStormDoc'
 
+
+def previous_version(ver):
+    # XXX: on incrementing major version, minor version counter is lost!
+    major, minor = ver.split('.')
```

Comment:
True, we need to drop the patch level.  
`ver.split(',',2)`  would work I assume.